src/org/jitsi/impl/neomedia/RTPConnectorOutputStream.java

```diff
@@ -663,6 +694,23 @@ private boolean write(RawPacket[] pkts)
         return success;
     }
 
+    /**
+     * @return the current output bitrate in bits per second.
+     */
+    public long getOutputBitrate()
```

Comment:
OutputBitrate of OutputStream... I think the Output on Bitrate is obsolete but I suppose getBitrate could be viewed as too sparse, vague, etc.